src/core/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const ApiService = {
```

Comment:
This ApiService object is not loosely coupled with the implementation details (axios could not be easily swapped with a different ajax service).
Consider a more general approach.